src/createR.js

```diff
@@ -4,40 +4,87 @@ const initialState = {
   status: STATUS.IDLE,
   loading: false,
   error: null,
-  data: null
+  data: null,
 }
 
-const createR = name => (state = initialState, action) => {
-  const { type, payload, meta } = action
+const defaultReducer = (state, action) => {
+  const { type, payload } = action
```

Comment:
Could not use unstructuring of these elements as function parameters?
`cosnt defaultReducer  = (state, { type, payload }) => ...`